compose/app/src/main/res/drawable-v24/ic_launcher_foreground.xml

```diff
@@ -0,0 +1,30 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Did compose remove the previous app launcher icon?

Reply:
Am still seeing the generic android launcher icon instead of the Chartboost Mediation one.